hack/verify-links-doc.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+#
+# This script is used to verify links in markdown docs.
+#
+# See https://docs.npmjs.com/resolving-eacces-permissions-errors-when-installing-packages-globally if you meet permission problems when executing npm install.
+
+ROOT=$(unset CDPATH && cd $(dirname "${BASH_SOURCE[0]}")/.. && pwd)
+cd $ROOT
+
+npm install markdown-link-check@3.8.1
+
+VERBOSE=${VERBOSE:-}
+CONFIG_TMP=$(mktemp)
+ERROR_REPORT=$(mktemp)
+
+trap 'rm -f $CONFIG_TMP $ERROR_REPORT' EXIT
+
+# Check all directories starting with 'v\d.*' and dev.
+for d in zh en; do
```

Comment:
There are no folders named `zh` and `en`, so we need to add the for loop here.

Reply:
done